chainer/functions/utils/forget.py

```diff
@@ -59,4 +59,59 @@ def backward(self, inputs, grads):
 
 
 def forget(func, *xs):
+    """Call a function without storing internal results.
+
+    On a forward-propagation Chainer stores all internal results of
+    :class:`Function` on a computational graph as they are required on
+    backward-propagation. These results consume too many memories. This method
```

Comment:
"""These results consume too many memories.""" It is better to note that this happens only if the internal result is too large?